Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertWall.cs

```diff
@@ -115,40 +117,22 @@ public List<ApplicationPlaceholderObject> WallToNative(BuiltElements.Wall speckl
 
       }
       else // Set wall unconnected height.
-      {
         TrySetParam(revitWall, BuiltInParameter.WALL_USER_HEIGHT_PARAM, speckleWall.height, speckleWall.units);
-      }
 
       SetInstanceParameters(revitWall, speckleWall);
 
-      var placeholders = new List<ApplicationPlaceholderObject>()
-      {
-        new ApplicationPlaceholderObject
-        {
-        applicationId = speckleWall.applicationId,
-        ApplicationGeneratedId = revitWall.UniqueId,
-        NativeObject = revitWall
-        }
-      };
-
-      var hostedElements = SetHostedElements(speckleWall, revitWall);
-      placeholders.AddRange(hostedElements);
-
-
-      Report.Log($"{(isUpdate ? "Updated" : "Created")} Wall {revitWall.Id}");
```

Comment:
Any specific reason why was this removed? Didn't check other conversions, but if possible we should keep the old Report in place....

Reply:
The new `ApplicationObject` status indicates if the element is updated or created! Unless you don't want me to include the new report for the revit connector?